Native SimNet.md

```diff
@@ -0,0 +1,227 @@
+We recommend to use the [xud-docker](User%20Guide.md) setup to trade on the `xud-simnet`; it does not require installation of `xud` or any dependencies. For a native installation, installing various dependencies of underlying clients, follow the guide below.
+
+## Goal
+
+This guide describes how to setup `xud` and connect to and trade on the XUD Simulation Network (`xud-simnet`). It should take about 15 minutes to complete all steps.
+
+`xud` is in alpha stage and the `xud-simnet` aims to give interested individuals an early 'look & feel' of how things will work lateron with a convenient, mostly automated setup. The `xud-simnet` uses magic simnet coins at all times.
```

Comment:
'look & feel' of how things will work ~~lateron~~ later on with a convenient

Reply:
Fixed